### PR TITLE
Blog updates

### PR DESCRIPTION
**Pull request by ibgreen**

### CODE REVIEW

File: _posts/2017-03-31-introducing-deckgl-v4.md [hunk 1]
```diff
@@ -13,36 +13,27 @@ We've just released a new major version of deck.gl, so this is a good time to sh
...
 
-Therefore, in this v4 release, the stars are the new layers enlisted in our deck.gl layer catalog that do magic with data. Now, the numder of core layers in deck.gl has grown to over a dozen and all of them are carefully audited, tested and documented [layers](https://uber.github.io/deck.gl/#/documentation/layer-catalog).
+In deck.gl v4, the core layer catalog in deck.gl effectively doubles in size to over a dozen carefully audited, tested and documented [layers](https://uber.github.io/deck.gl/#/documentation/layer-catalog), covering a significantly wider array of geospatial visualization use cases.
+
+Obviously, while the new layers are the big stars of the release, deck.gl v4 includes more than just the layers. It covers almost five months of intensive development, during which have received and addressed many good comments, suggestions, bug reports and feature requests, both from internal and external users. If you are already a deck.gl v3 user, we believe that as you start working with v4, you will find few areas, whether in terms of APIs, documentation, examples or actual code, that haven't received some level of additional polish.
```
Comment (shaojingli) [inline _posts/2017-03-31-introducing-deckgl-v4.md]: > It covers almost five months of intensive development, during which have received and addressed many good comments,

The subject is "deck.gl v4". It cannot receive and address anything. 

Recommendation: Within almost five months of intensive development, we have received...

File: _posts/2017-03-31-introducing-deckgl-v4.md [hunk 2]
```diff
@@ -93,12 +84,11 @@ The `PointCloudLayer` is designed to visualize 3D point cloud data, usually gene
...
+In this deck.gl release, we have added two layers of the "auto-aggregating" type that was introduced with `ScreenGridLayer` in deck.gl v3. These layers go beyond simply displaying user data on the map. The `GridLayer` and `HexagonLayer` draw equal-area rectangular or hexagonal cells. However, they don't draw one cell per data point like most deck.gl layers do. They first aggregate (or "bin") the user-provided data points into these cells, and then draw the cells using the aggregated values, effectively rendering grid-based **heatmaps**.
```
Comment (shaojingli) [inline _posts/2017-03-31-introducing-deckgl-v4.md]: > we have added

This is the only place in the article that we use past perfect tense

File: _posts/2017-03-31-introducing-deckgl-v4.md [hunk 3]
```diff
@@ -155,14 +144,14 @@ We have already started experimenting with the powerful features in the newly re
...
 
-### More Advanced Picking (Selection) Support
+### More Advanced Picking Support
 
-Picking or selection is also one of the most common tasks of any data visualization applications and is a core functionality we provide with deck.gl. In the v4 release, a color-coded picking process that are both much more robust and performant is implemented. We plan to continue improving picking by adopting better picking boundary arbitration, supporting square or polygon area picking, etc.
+Picking (e.g. using a mouse to select the graphical representation of one data point to highlight it or show more information about it) is one of the most common tasks of any data visualization applications and is a core functionality we provide with deck.gl. We have ideas for improving picking by adopting better picking boundary arbitration (bigger pick targets), supporting square or polygon area picking, etc.
```
Comment (shaojingli) [inline _posts/2017-03-31-introducing-deckgl-v4.md]: I believe "Mouse picking or mouse selection" is a self-explanatory. But if you think it's better to be more verbose. I'm okay with it.

File: _posts/2017-03-31-introducing-deckgl-v4.md [hunk 4]
```diff
@@ -13,36 +13,27 @@ We've just released a new major version of deck.gl, so this is a good time to sh
 
 ### What Motivated the Release?
 
-Since the big deck.gl v3 release in November 2016, deck.gl has seen rapid adoption across a number of internal data visualization applications at Uber. We have also received many good comments, suggestions and requests, both from internal and external users. We noticed that developers are not only building apps using deck.gl's highly performant layers, but also starting to unleash their own creativity by creating various new layers. This is pushing deck.gl into areas that we have never thought it would be.
+When we did our first external [announcement](https://eng.uber.com/deck-gl-framework/) of deck.gl in November 2016 (announcing the deck.gl v3 release), we also stated our intent to improve and expand deck.gl's layer catalog in future releases, and that is a major part of what we are doing now through the deck.gl v4 release.
```
Comment (shaojingli) [inline _posts/2017-03-31-introducing-deckgl-v4.md]: [external announcement] would be better

**Comment by ibgreen** [comment]
@shaojingli Thanks, good catches, all are addressed. Merging.

**Pull request merged**